### ISSUE TITLE
docker-compose up --force-recreate throws exception

### ISSUE THREAD

**Issue by fkautz** [docker/compose#2385]
```sh
$ docker-compose up --force-recreate
```

```py
Recreating reproduciblebuilds_redis_1
Recreating 589ef0f368_589ef0f368_reproduciblebuilds_repro_1
Traceback (most recent call last):
  File "/usr/lib64/python2.7/logging/__init__.py", line 859, in emit
    msg = self.format(record)
  File "/usr/lib64/python2.7/logging/__init__.py", line 732, in format
    return fmt.format(record)
  File "/usr/lib/python2.7/site-packages/compose/cli/formatter.py", line 48, in format
    return self.get_level_message(record) + message
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 124: ordinal not in range(128)
Logged from file main.py, line 66
```

**Comment by dnephin** [comment]
Please provide the output of:

* `docker-compose version`
* the command you used to install compose (curl, pip, other)

From the output I'm guessing you are not using the latest version 1.5.0. We've fixed a bunch of these unicode errors in the 1.5.0 release.

**Comment by dnephin** [comment]
Closing, not enough information, and probably a duplicate of #3817
